tools/build.ts

```diff
@@ -22,40 +21,13 @@ const buildBundle = process.argv.indexOf('--build-bundle') !== -1;
 // Path variables
 const ROOT = path.join(__dirname, '..');
 const DEPS = path.join(ROOT, 'deps');
-const DEPS_EIGEN = path.join(DEPS, 'eigen');
-const DEPS_EMSDK = path.join(DEPS, 'emsdk');
-const DEPS_EMSDK_EMSCRIPTEN = path.join(DEPS_EMSDK, 'upstream', 'emscripten');
 const DEPS_ONNX = path.join(DEPS, 'onnx');
-const EMSDK_BIN = path.join(DEPS_EMSDK, 'emsdk');
-const SRC = path.join(ROOT, 'src');
-const SRC_WASM_BUILD_CONFIG = path.join(SRC, 'wasm-build-config.json');
 const TEST = path.join(ROOT, 'test');
 const TEST_DATA = path.join(TEST, 'data');
 const TEST_DATA_NODE = path.join(TEST_DATA, 'node');
 const OUT = path.join(ROOT, 'dist');
-const OUT_WASM_JS = path.join(OUT, 'onnx-wasm.js');
-const OUT_WASM = path.join(OUT, 'onnx-wasm.wasm');
-
-// Emcc (for Wasm) compile flags
-// Add new compiler flags here (if needed)
-const BUILD_OPTIONS = [
-  '-I' + DEPS_EIGEN,
-  '-DEIGEN_MPL2_ONLY',
-  '-std=c++11',
-  '-s WASM=1',
-  '-s NO_EXIT_RUNTIME=0',
-  '-s ALLOW_MEMORY_GROWTH=1',
-  '-s SAFE_HEAP=0',
-  '-s MODULARIZE=1',
-  '-s SAFE_HEAP_LOG=0',
-  '-s STACK_OVERFLOW_CHECK=0',
-  // '-s DEBUG_LEVEL=0', // DEBUG_LEVEL is disabled in emsdk 1.39.16
-  '-s VERBOSE=0',
-  '-s EXPORT_ALL=0',
-  '-o ' + OUT_WASM_JS,
-  '-O2',
-  '--llvm-lto 3',
-];
+const OUT_WASM_JS = path.join(OUT, 'onnxruntime_wasm.js');
+const OUT_WASM = path.join(OUT, 'onnxruntime_wasm.wasm');
```

Comment:
Hi, for "onnxruntime_wasm_threads" we will need to change the file names here. Thanks.